docs/pages/management/guides/docker.mdx

```diff
@@ -149,10 +151,104 @@ The Web UI will be available at the displayed URL.
 
 (!docs/pages/includes/insecure-certificate.mdx!)
 
-## Step 4/4. tsh into your Teleport container
+## Step 4/4. Try Server Access
 
-After you have finished creating your user, open a second terminal and issue the
-command, which will log in to your Teleport cluster via the Proxy Service at
+This docker installation includes just the Teleport Proxy and Auth Service.  You can try
+accessing a server through Teleport by installing the Teleport SSH Service
+on a Ubuntu docker container. Here are the steps.
+
+First start a Ubuntu image:
+```code
+$ docker run -it --name=example-server --hostname=example-server ubuntu:latest bash
+```
+
+<Tabs>
+<TabItem scope={["oss"]} label="Open Source">
```

Comment:
Could we include `curl` instructions in one snippet, then include the `install-linux.mdx` partial?

Reply:
The install linux includes `sudo`, we'd have to include that too to install.  Plus the install linux includes multiple types.